projects/graphql-client/src/graphql-config.ts

```diff
@@ -47,3 +46,9 @@ export const enum GraphQlHandlerType {
   Mutation = 'mutation',
   Query = 'query'
 }
+
+export interface GraphQlOptions {
+  uri: string;
+  batchSize?: number;
+  batchDuration: number;
```

Comment:
misleading name - maybe `batchDebounceTimeMs` ? Also doesn't look to be required since we're providing a default.

Reply:
cool